packages/taco/README.md

```diff
@@ -60,7 +60,7 @@ const decryptedMessage = await decrypt(
   web3Provider,
   domains.TESTNET,
   messageKit,
-  getPorterUri(domains.TESTNET),
+  await getPorterUri(domains.TESTNET),
```

Comment:
Would this work - `getPorterUri` only returns a `string` and not `string[]`. Or does typescript automatically handle that gracefully?

Reply:
would it be ok to remove that line? basically it will show how to use default URIs

Reply:
👍 - sounds reasonable to me.